intro-Cloud/ch_Flavors.tex

```diff
@@ -91,44 +93,60 @@ \section{Flavor Sizes}\label{sec:flavor-sizes}
 
 \begin{table}[h!]
 \centering
-\begin{tabular}{ |p{3cm}|p{3cm}|p{3cm}|p{3cm}| }
+\begin{tabular}{ |p{3cm}|p{3cm}|p{3cm}|p{3cm}|p{3cm}| }
   \hline
-  \rowcolor{lightgray} \textbf{Flavor Size} & \textbf{RAM} & \textbf{Root Disk} & \textbf{vCPUs} \\
+  \rowcolor{lightgray} \textbf{Flavor name} & \textbf{RAM} & \textbf{Root Disk} & \textbf{\# vCPUs} & \textbf{\# vGPUs} \\
```

Comment:
not sure if we should include vgpus in this table, its just true or false, an extra appliance like ups, we do not have quota for vgpus either.